arch/arm/boot/dts/overlays/cm-swap-i2c0-overlay.dts

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-2.0-only
+// Definitions for IMX708 camera module on VC I2C bus
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/gpio/gpio.h>
+
+/{
+	compatible = "brcm,bcm2835";
+
+	i2c0mux_frag: fragment@0 {
```

Comment:
I think this label is in the wrong place - it should be on the `__overlay__` node, otherwise the changes fall down the cracks.

Reply:
I'll have to check that tomorrow then. I was copying https://github.com/raspberrypi/linux/blob/rpi-5.15.y/arch/arm/boot/dts/overlays/imx219-overlay.dts#L18

Reply:
```
$ ovmerge arch/arm/boot/dts/overlays/cm-swap-i2c0-overlay.dts,i2c0-gpio44
/dts-v1/;
/plugin/;

#include <dt-bindings/gpio/gpio.h>

/ {
	compatible = "brcm,bcm2835";
	fragment@0 {
		target = <&i2c0mux>;
		pinctrl-0 = <&i2c0_gpio44>;
		__overlay__ {
			pinctrl-0 = <&i2c0_gpio28>;
			pinctrl-1 = <&i2c0_gpio0>;
		};
	};
};
```

Reply:
As expected, you were right. Now fixed and tested on CM3 with i2c10-gpio44 and rerouting the I2C.